classes/class-wc-connect-taxjar-integration.php

```diff
@@ -154,6 +165,33 @@ public function add_tax_settings( $tax_settings ) {
 		return $tax_settings;
 	}
 
+	/**
+	 * Hack to force keepAlive: true on tax setting tooltip.
+	 */
+	public function fix_tooltip_keepalive() {
+		global $pagenow;
+		if ( 'admin.php' !== $pagenow || ! isset( $_GET['page'] ) || 'wc-settings' !== $_GET['page'] || ! isset( $_GET['tab'] ) || 'tax' !== $_GET['tab'] || ! empty( $_GET['section'] ) ) {
+			return;
+		}
+
+		$tax_settings = $this->add_tax_settings( array() );
```

Comment:
If all we need is `desc_tip`, we could refactor the `desc_tip` out into its own function:
```
function getTaxRateTooltip() {
$store_settings = $this->get_store_settings();
	$all_states = WC()->countries->get_states( $store_settings['country'] );
	$full_state = $all_states[ $store_settings['state'] ];
	$descTip = sprintf( __( 'Your tax rates and settings will be automatically configured for %1$s. <a href="https://docs.woocommerce.com/document/setting-up-taxes-in-woocommerce/#section-12">Learn more about setting up tax rates for additional nexuses</a>', 'woocommerce-services' ), $full_state );
    return $descTip;
}
```

Then we don't need to worry about any side effects that may be caused by `add_tax_settings()` function. But I took a look at `add_tax_settings()`, looks like it's a function that transform input to certain output, it doesn't modify any global variables/states. I think this is fine as well.